### PR TITLE
NAS-127418 / 24.10 / Add readonly label to edit forms

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Testing:
See ticket & forms

### CODE REVIEW

**Review by denysbutenko — COMMENTED** [review]
The `readonly` badge appears when logged as a root:
image

The `Save` button is enabled when logged as a readonly:
image

I keep testing

**Review by denysbutenko — COMMENTED** [review]
image
image

**Comment by AlexKarpov98** [comment]
@denysbutenko - good catch, I fixed it now.

**Comment by denysbutenko** [comment]
I just realized the scope of the ticket. Sorry for the non-related bugs.

File: src/app/modules/ix-forms/components/ix-slide-in/components/ix-modal-header/ix-modal-header.component.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
 >
   <h3 class="ix-formtitle">
     {{ title | translate }}
+    <ix-readonly-badge *ngIf="requiredRoles?.length && !(hasRequiredRoles | async)"></ix-readonly-badge>
```
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply src/app/modules/ix-forms/components/ix-slide-in/components/ix-modal-header/ix-modal-header.component.html:8]: > ix-modal-header2

yes!

**Review by undsoft — APPROVED** [review]
Looks good.

Some issues that I've noticed:
- Worth adding Readonly label to Groups -> Members.
- Custom App button can be disabled.
- Credentials -> Directory services has misaligned icons on two Configure buttons and weird lock icons in idmap table.
Снимок экрана 2024-02-27 в 10 46 12

- In datasets lock icons get misaligned on some resolutions.
Снимок экрана 2024-02-27 в 10 39 20

**Comment by AlexKarpov98** [comment]
@undsoft @denysbutenko - thanks
I gonna fix it ASAP 😄

**Pull request merged**